Bottom: 0ed1bf83a546ea60e3c59b36fcefd3dcfeef8358
Top:    de505746938d37d0abc4632e50d3b45159037177
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 15:11:31 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 15b3667..2db68e8 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -962,8 +962,6 @@ ghid_drawing_area_configure_hook (GHidPort *port)
 {
 }
 
-static hidGC savefg = NULL;
-
 gboolean
 ghid_start_drawing (GHidPort *port)
 {
@@ -977,8 +975,6 @@ ghid_start_drawing (GHidPort *port)
 
   port->render_priv->in_context = true;
 
-  savefg = Output.fgGC;
-
   Output.fgGC = gui->make_gc ();
   Output.bgGC = gui->make_gc ();
   Output.pmGC = gui->make_gc ();
@@ -1006,7 +1002,7 @@ ghid_end_drawing (GHidPort *port)
   gui->destroy_gc (Output.bgGC);
   gui->destroy_gc (Output.pmGC);
 
-  Output.fgGC = savefg;
+  Output.fgGC = NULL;
   Output.bgGC = NULL;
   Output.pmGC = NULL;
 }
